app/components/Comments/CommentDisplay.jsx

```diff
@@ -80,25 +81,25 @@ export class CommentDisplay extends React.PureComponent {
                 }
                 {source && <Source withoutPlayer={!richMedias} source={source}/>}
               </div>
-              {!withoutActions &&
+              {!withoutActions && (
                 <nav className="comment-actions">
-                  { isOwnComment ? this.renderOwnCommentActions() : this.renderOtherCommentActions()}
-                  { replies &&
-                    <a className={!showReplies ? "reply-collapsed" : null}
-                       onClick={() => this.setState({showReplies: !showReplies})}>
-                      <Icon size="small" name={showReplies ? "eye-slash" : "eye"}/>
-                      <span>
-                        {t('videoDebate:comment.replies', {
-                          context: showReplies ? 'hide' : 'show',
-                          count: replies.size
-                        })}
-                      </span>
-                    </a>
+                  { isOwnComment
+                    ? this.renderOwnCommentActions()
+                    : this.renderOtherCommentActions()
                   }
+                  { replies && this.renderAction(
```

Comment:
maybe this `renderAction` call could be extract of the JSX code,
there is (too) much (?) logic in it

Reply:
Done